components/FeaturedSlider.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Link from "next/link";
+import Poster from "./Poster";
+
+// Import Swiper React components
+import { Swiper, SwiperSlide } from "swiper/react";
+
+// Import Swiper styles
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
+import "swiper/css/scrollbar";
+
+// import required modules
+import { Pagination } from "swiper";
+
+function FeaturedSlider({ gameProp, sliderTitle = "" }) {
+  console.log("---------Game Prop------");
```

Comment:
We should avoid merging comments into main, just so we aren't confused by debug console logs in the future. When we start running 'production' level code, we should also avoid any console logs to hide debug code from the user.